model-optimizer/extensions/middle/LayoutChangeForEinsum.py

```diff
@@ -31,27 +39,33 @@ def find_and_replace_pattern(self, graph: Graph):
             connected_in_ports = [port for port in einsum.in_ports().values() if not port.disconnected()]
             num_inputs = len(connected_in_ports)
 
+            # check if correct_data_layout attribute is set for inputs and output
+            input_correct_layout_mask = []
+            for input_ind in range(num_inputs):
+                input_correct_layout_mask.append(is_input_data_in_correct_layout(einsum, input_ind))
+            output_correct_layout_mask = is_output_data_in_correct_layout(einsum, 0)
```

Comment:
nit: the `output_correct_layout_mask` is not actually a mask. It is a single value. Please, rename.

Reply:
Done.